calculator/main.py

```diff
@@ -58,18 +89,23 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history'] 
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
 
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    if calc['history']:
+        history = calc['history'][len(calc['history'])-1]
```

Comment:
You can get the last entry in a list by using an index of `-1` eg: `my_list[-1]`
You can also directly index the output of an indexable item if it is in turn indexable, so you could simplify these two lines by using `return calc['history'][-1][-1]`